pytest_docker_tools/factories/container.py

```diff
@@ -8,16 +8,49 @@
 def container(request, docker_client, wrapper_class, **kwargs):
     """ Docker container: image={image} """
 
+    wrapper_class = wrapper_class or Container
+
+    if request.config.option.reuse_containers:
+        if "name" in kwargs.keys():
+            name = kwargs["name"]
+            current_containers = docker_client.containers.list(ignore_removed=True)
+            for cont in current_containers:
+                if cont.name == name:
+                    if (
+                        "Pytest-Docker-Tools.Reusable_Container"
+                        in cont.attrs["Config"]["Labels"]
+                        and cont.attrs["Config"]["Labels"][
+                            "Pytest-Docker-Tools.Reusable_Container"
+                        ]
+                        == "True"
+                    ):
+                        return wrapper_class(cont)
+        else:
+            raise RuntimeError(
```

Comment:
Would pytest's `UsageError` work here or is it already too late in execution?

Reply:
UsageError is working.